src/worldcereal/openeo/feature_extractor.py

```diff
@@ -110,6 +130,12 @@ def execute(self, inarr: xr.DataArray) -> xr.DataArray:
             get_presto_features,
         )
 
+        slope = self._compute_slope(inarr.isel(t=0))
```

Comment:
I'm wondering why you chose to compute slope in the Presto feature extractor. I guess there's no other way to do it, like in where we actually assemble inputs? That's maybe not a UDF? It just feels a bit weird that slope computation is part of presto feature extractor.

Reply:
Also, this means each time we run presto feature extractor (by default twice for cropland vs croptype, but even many more times in developing the algorithms locally), slope has to be recomputed. Would be nicer to have slope as part of the preprocessed inputs. But so I guess it won't be possible in an efficient way?

Reply:
There's no process that does it. We could create a brand new UDF just for this aspect but it would be a heavier task. Also  not sure that it's very efficient. Also when you think about it other features are computed within this UDF from the Presto library such as lat/lon